### DISCUSSION TITLE
Would this work in HP Elitebook 840 G2

### DISCUSSION THREAD

**Discussion by BokaManush** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#154] [Q&A]
I need battery status fixing

**Comment by chris1111** [comment]
Elitebook 840 G2 not support here

**Comment by BokaManush** [comment]
Okay Thank You

**Comment by budgierless** [comment]
> Elitebook 840 G2 not support here

Hi, I have used your work before for my other laptop, thanks for all you do.
I now have an HP EliteBook-840 G4 and my question is why is this device not supported as it seem is first post in this thread manged toget a working system with a little issue. The main reason I ask is only so I canbe aware of problems you know of that i wont find out about at first glance? maybe a hardware issue or only old OS problem, please advice. as I only found information with [[HP EliteBook 840 G4](https://github.com/XXCoreRangerX/Hackintosh-HP-EliteBook-840-G4)] so far.

**Comment by chris1111** [comment]
> Hi, I have used your work before for my other laptop, thanks for all you do.
> I now have an HP EliteBook-840 G4 and my question is why is this device not supported as it seem is first post in this thread manged toget a working system with a little issue. The main reason I ask is only so I canbe aware of problems you know of that i wont find out about at first glance? maybe a hardware issue or only old OS problem, please advice. as I only found information with [[HP EliteBook 840 G4](https://github.com/XXCoreRangerX/Hackintosh-HP-EliteBook-840-G4)] so far.

Its here for your laptop [EliteBook 840 G4](https://github.com/chris1111/HP-EliteBook-840-G4) If its not support in the program its about DSDT compiller issue and DSDT patch

**Reply by budgierless** [reply]
Thanks so much, you are a legend as always.
